src/libraries/System.Net.WebSockets.Client/src/System/Net/WebSockets/ClientWebSocketOptions.cs

```diff
@@ -46,6 +48,8 @@ public Version HttpVersion
             }
         }
 
+        /// <summary>Gets or sets the policy that determines how <see cref="P:System.Net.WebSockets.ClientWebSocketOptions.HttpVersion" /> is interpreted and how the final HTTP version is negotiated with the server.</summary>
```

Comment:
Since this is a nested type, I _think_ this should work. Can you please confirm?:
```suggestion
        /// <summary>Gets or sets the policy that determines how <see cref="ClientWebSocketOptions.HttpVersion" /> is interpreted and how the final HTTP version is negotiated with the server.</summary>
```